langchain/src/base_language/index.ts

```diff
@@ -177,6 +177,47 @@ export abstract class BaseLanguageModel
     return numTokens;
   }
 
+  async getNumTokensFromMessages(messages: BaseMessage[]): Promise<{
```

Comment:
Do we need to move this in here? Can we just reuse this?

https://github.com/hwchase17/langchainjs/blob/2e63ee0a4e685b43f420d79d7c38ba6d16e374fa/langchain/src/chat_models/openai.ts#L501

Reply:
ok~

Reply:
Sorry actually let's just use the more generic method I changed it to - this one will only handle OpenAI